testcontainers/mockserver.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any
+
+import requests
+from mockserver import MockServerClient
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready
+
+
+class MockServerContainer(DockerContainer):
+    """
+    MockServer container.
+
+    Example
+    -------
+    ::
+
+        with MockServerContainer() as mockserver:
+            mockserver_client = mockserver.get_client()
+            mockserver_client.stub(request(method='GET', path='/path'),
+                                   response=response(code=HTTPStatus.OK))
+
+            response = requests.get(f'{mockserver.base_url}/path')
+            assert response.status_code == HTTPStatus.OK
+
+            mockserver_client.verify()
+    """
+
+    EDGE_PORT = 1080
+    IMAGE = 'mockserver/mockserver:latest'
+
+    def __init__(self, image: str = IMAGE, **kwargs: Any):
```

Comment:
Would it be better to make use of the port argument so one can overwrite the port? 
Since the port can be changed with an environment variable: https://www.mock-server.com/where/docker.html#docker_container_environment_variables